department-of-veterans-affairs/vets-api
# [38358] Add logingov to backend_statuses

## Summary

- Add logingov pagerduty ID to settings
- [PagerDuty service](https://dsva.pagerduty.com/service-directory/P2SHMM9/integrations?welcome=1) created
- [Datadog Monitor](https://vagov.ddog-gov.com/monitors/120105) Created

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#38358

## Testing done
- start server - `rails s`
- Add valid PagerDuty api_key to `settings.local.yml`
- `GET localhost:3000/v0/backend_statuses`
- Observe:
```json
{
  "service": "Login.gov",
  "service_id": "logingov",
  "status": "active",
  "last_incident_timestamp": null
}
```

## What areas of the site does it impact?
backend_statuses, monitoring/alerts

## Acceptance criteria


## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
